api/src/main/java/org/eclipse/microprofile/health/Startness.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2021 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICES file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+package org.eclipse.microprofile.health;
+
+import javax.enterprise.util.AnnotationLiteral;
+import javax.inject.Qualifier;
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/**
+ *
+ * This qualifier is used to define a Startness Health Check procedure mapped to the Startup Kubernetes probes.
+ *
+ * @author Martin Stefanko
+ * @since 3.1
+ */
+@Target({ TYPE, METHOD, PARAMETER, FIELD })
+@Retention(RUNTIME)
+@Documented
+@Qualifier
+public @interface Startness {
```

Comment:
I don't think the word `Startness` to define the new health check procedure make sense here. I think we should call it `Startup` to be consistent with K8s.

Reply:
This is an intentional decision to mainly not introduce any confusion with https://javaee.github.io/javaee-spec/javadocs/javax/ejb/Startup.html as we are targeting Java EE developers. Also, it stays aligned with the other annotations. I understand this concern, it also took me a lot of thinking and internal discussions but I was convinced that Startness would be better. Please also note that this was introduced in our implementation a while back and I haven't heard any concerns from the community about the name.

Reply:
Right, that makes sense! Totally forgot about the EJB Startup bean. Since, the name is consistent with the existing health check procedures, I think we should be good.

Reply:
Hello @tjquinno. Thank you for your feedback. I understand this is not grammatically correct but after weighing all arguments this came to us as the best alternative.

The discussions I mentioned happened in our implementation way before we proposed this functionality to the specification. This is why they were internal. Startness was provided in our implementation for several months now.

Personally, I still prefer Startness because the conflict with Startup is more important to me but this specification is a collaborative effort and it's not my place to discard any other opinions. However, as you've mentioned the plan ballot is ending today and I planned to release RC1 tomorrow. Going into this discussion now would most likely delay the release of Health 3.1 and in turn, also delay the release of MP 4.1. With this in mind, I would like to continue with the current naming and if we run into issues with the client adoption (note that I didn't receive this kind of feedback in our implementation up to this day) we will change it in Health 4.0 with most likely MP 6.x. Would this be acceptable?

Reply:
Of course, that's fine with me. Commenting at this point I never suggested nor did I expect the wheels of progress to stop turning on Health 3.1 or MP 4.1. Even so, I did want to explain my thinking, and I appreciate you accepting it in the collaborative spirit I intended.

Reply:
@tjquinno at the moment, 3.1-RC1 was cut. However it is not too late to make more changes. This 3.1RC1 was released to get more feedback. As for this name, either `Startup` or `Startness` has drawbacks. How about `StartupCheck`? Though it is not same as startup probe but it avoids the name clashes with ejb startup and it is closer to the startup probe than the name `Startness`, which I also found difficult to comprehend.

Reply:
Yes, we have reopened the naming discussion because there is a bigger pushback. But please keep your responses in the GG thread - https://groups.google.com/g/microprofile/c/j0f2zRBBMPU/m/LpQGlDqwAAAJ.